packages/common-server/src/etc.ts

```diff
@@ -14,6 +16,31 @@ export class NodeJSUtils {
   }
 }
 
+/**
+ * Various utilities that are not categorized
+ */
+export class EtcUtils {
```

Comment:
How about something like `RuntimeUtils` or maybe bundled with the `Environent` stuf? (`getStage`) etc.?

Reply:
done